app/Models/Product_order.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
it is not common to make models for many to many pivot tables.

if you don't need it, please remove it.
if you need, it's not good and we should talk about the usage.